client/src/views/statistics/statistics.module.scss

```
@@ -0,0 +1,62 @@
+@import '../../core/styles/color.scss';
+
+.statistics-page {
+  position: absolute;
+  width: 100%;
+  top: 172px;
```

```suggestion
  position: relative;
  width: 100%;
  top: -32px;
```

`<main>`의 style을 global.scss에서 `position: relative` 으로 만들고 page를 상대적으로 올리는게 어떨까 합니다.
다른 페이지들도 상단 기준으로 돼 있는데 반응형에서 틀어질것 같습니다.